_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -46736,6 +46724,11 @@
 /obj/effect/turf_decal/tile/red{
 	dir = 8
 	},
+/obj/machinery/atmospherics/components/binary/pump{
+	dir = 1;
+	name = "N2 to Airmix";
+	on = 1
```

Comment:
use /obj/machinery/atmospherics/components/binary/pump/on instead, if you dont it'll look like it's turned off in the map editor

Reply:
Fixed, thanks.